introduction.md

```diff
@@ -1 +1,9 @@
 Hi, I'm Daniel and I think the Ruby path is really good
+
+Deadly Skills:
+* iOS
+* Git
+* HTML
+* CSS
+* JavaScript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?